test/unit/src/common/reducers/entities/t_repository.js

```diff
@@ -0,0 +1,66 @@
+import expect from 'expect';
+
+import repository from '../../../../../../src/common/reducers/entities/repository.js';
+import * as fixtures from './fixtures.js';
+
+describe('repository entities', function() {
+
+  context('update entity reducer', function() {
+
+    let state;
+
+    beforeEach(function() {
+      state = repository(undefined, {
+        type: 'REPO_ADD',
+        payload: {
+          id: 1001
```

Comment:
I don't like `1001` being hardcoded all over the place instead of imported from fixtures or similar, but it doesn't matter that much now.

But we should consider making it into variable or constant imported from fixture later.